package.json

```diff
@@ -44,5 +46,25 @@
     "ts-node": "~7.0.0",
     "tslint": "~5.11.0",
     "typescript": "~3.2.2"
+  },
+  "build": {
+    "builder": "@angular-devkit/build-angular:browser",
+    "options": {
+      "outputPath": "dist/ng6",
+      "index": "src/index.html",
+      "main": "src/main.ts",
+      "polyfills": "src/polyfills.ts",
+      "tsConfig": "src/tsconfig.app.json",
+      "assets": [
+        "src/favicon.ico",
+        "src/assets"
+      ],
+      "styles": [
+        "src/styles.css","node_modules/bootstrap/dist/css/bootstrap.min.css"
+
+      ],
+      "scripts": ["node_modules/jquery/dist/jquery.min.js",
+                 "node_modules/bootstrap/dist/js/bootstrap.min.js"]
+    }
```

Comment:
Please remove `build`'s chunk of code and add `styles` prop to `angular.json` config